src/System.Text.Primitives/System/Text/Formatting/Internal/TimeFormatter.Utf8.cs

```diff
@@ -0,0 +1,315 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Text
+{
+    internal static class InvariantUtf8TimeFormatter
```

Comment:
Aren't these parts of the methods exactly the same (aside from the difference that one takes a ref char while the other takes ref byte)?

https://github.com/shiftylogic/corefxlab/blob/9b46b6344f2feae9e4ffdff1fc074ec42203dad3/src/System.Text.Primitives/System/Text/Formatting/Internal/TimeFormatter.Utf8.cs#L76-L113

https://github.com/shiftylogic/corefxlab/blob/9b46b6344f2feae9e4ffdff1fc074ec42203dad3/src/System.Text.Primitives/System/Text/Formatting/Internal/TimeFormatter.Utf16.cs#L78-L115